docs/guide/burst_analysis.md

```diff
@@ -0,0 +1,171 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Burst Analysis
+
+Bursting is defined as the occurence of a specified number of simultaneuos spikes (usually >10), with a small interspike interval (usually < 100ms) [1,2]
+
+References: 
+
+[1] Chiappalone, Michela, et al. "Burst detection algorithms for the analysis of spatio-temporal patterns in cortical networks of neurons." Neurocomputing 65 (2005): 653-662.\
```

Comment:
https://www.markdownguide.org/basic-syntax/#reference-style-links

Please change according to these.

Reply:
commit 9d9220d68c331445cf902b1ab2923c3375f53ba8